src/toil/jobStores/abstractJobStore.py

```diff
@@ -1872,15 +1874,28 @@ class JobStoreSupport(AbstractJobStore, metaclass=ABCMeta):
     stores.
     """
 
+    ftp = None
```

Comment:
If we keep a class-level `FtpFsAccess` instance that is never destroyed, and the `FtpFsAccess` instance caches open FTP connections to servers, when do we close our FTP connections? It might be never, and that's probably not what we want.